docs/pages/eas-update/faq.mdx

```diff
@@ -18,6 +18,6 @@ EAS Update is a great way to quickly deliver improvements to the people who use
 
 ### Are Classic Updates still supported
 
-Expo is committed to supporting projects that use Classic Updates, which is the updates service available before December 2021. Apps that receive updates from the Classic Updates service will continue to work. With that, we are not planning on adding new features to Classic Updates other than fixing general issues and implementing security fixes as they arise.
+The Classic Updates service, which has been available before December 2021, is now deprecated. Users can no longer run `expo publish`, but existing apps will continue to receive Classic Updates that have already been published and are actively used.
```

Comment:
```suggestion
The Classic Updates service is now deprecated. It was available before December 202. You cannot run `expo publish` in newer projects. However, existing apps will continue to receive Classic Updates that have already been published and are actively used.
```

Let's add a distinction about the command no longer working in newer projects.

Reply:
We're turning off `expo publish` on Feb 12 for all projects, so it will apply to both old and new

Reply:
In that case, we can discard my comment. 

I'd still recommend swapping  "Users" with "You" since users are mostly developers here and its kind of implicit (updated the suggestion above).